meta/labs/meetings/15-09-03_who_writes_the_initial_playbook.md

```diff
@@ -0,0 +1,67 @@
+# Who Writes the Initial Draft of the Playbook?
+
+_Date:_
+Thursday, September 3rd, 2015
```

Comment:
Do you want these to be on separate lines? If so, you need an extra space between lines 3 and 4.